S3/views.py

```diff
@@ -122,3 +124,25 @@ def delete(self, request: Request, *args, **kwargs):
 
         s3.delete()
         return Response({'success': True, 'message': 'Successfully deleted!'}, status=status.HTTP_200_OK)
+
+
+@api_view(['GET'])
+@permission_classes([])
+def find_hash_by_combined_words(request: Request):
```

Comment:
> and there are only data of combined_words, not a func or method to use it,
> so you implement that and, related two data, named "combined_words", "hashed_value"?

I just implemented feature that finding hashed value using two combined words.
function that actually handles view is here

@ds1sqe

Reply:
yep

Reply:
if client send GET with two word, than, return a hashed value

Reply:
if there is data

Reply:
ㅇㅇ